.gitignore

```diff
@@ -1,3 +1,5 @@
 # DAppNodeSDK release directories
```

Comment:
Could you please add to the project eslint configuration? `.eslintr.json`. Take @dappnode/types as reference